src/app/store/selectors/VehicleTestResultModel.selectors.ts

```diff
@@ -3,6 +3,8 @@ import { IVehicleTestResultModelState } from '@app/store/state/VehicleTestResult
 import { TestRecordTestType } from '@app/models/test-record-test-type';
```

Comment:
can we please rename this file VehicleTestResult.selectors.ts instead
I am not sure why we have model at the end as it makes it confusing and this file is more a state selector.

Reply:
@JonLazariniDD should we be doing this now? I think we should have another ticket to update the naming convention.

Reply:
This is a new file we created it seems so we can do it.
If we need to agree on conventions we should set up a call but this seems to be straightforward to me.

`/selectors/<NameOfReducerFile>.selectors.ts`
Thoughts @alleypa @m-ujaffer @rspasici ?

Happy to use whatever makes sense with the team but having `Model` in a selector file makes it very confusing as this is not a Model but a selector.
For the remaining files, we can look individually to see what could be easy to change or not. Some of them might be quite straight forward, others less.

Reply:
yes agree to not have `Model`. I will suggest we do this in a dedicated ticket so that we don't distract the focus on this ticket.